src/main/scala/edu/ie3/simona/model/participant/PvModel.scala

```diff
@@ -141,7 +141,7 @@ final case class PvModel private (
       eTotal,
       data.dateTime,
       irraditionSTC
-    ) * scalingFactor
+    )
```

Comment:
Are we sure, that this needs to remove here? (I have to get into this as well again...)